src/redux/dashboard/reducer.js

```diff
@@ -0,0 +1,71 @@
+
+const SET_TOTAL_LHT_BALANCE_START = 'dashboard/SET_TOTAL_LHT_BALANCE_START'
+const SET_TOTAL_LHT_BALANCE_SUCCESS = 'dashboard/SET_TOTAL_LHT_BALANCE_SUCCESS'
```

Comment:
Action types constants should be prefixed with duck name, because they also should be exported from here (at least for specs)
Best option for this lines is:
`const DASHBOARD_TOTAL_LHT_FETCH = 'dashboard/TOTAL_LHT_FETCH'`
`const DASHBOARD_TOTAL_LHT = 'dashboard/TOTAL_LHT'`
Use this as example